src/composables/pnftInteractions.ts

```diff
@@ -156,3 +156,29 @@ export function generateTicketDetailLink(ticket: PNFT, appUrl?: string)  {
    return typeof attr != 'undefined' ? attr : "Attribute Not Set"
  }
 
+
+export function formatBylineTicketDatetime(ticketDatetimeString: string){
+   /* Input: Takes in a ticket's datetime value
+      Output: converts ticket's datetime value to specified string format
+      Example input: 2022-02-11T01:42:25.185Z
+      Example output: Thursday, Feb 10 2022 at 8:42PM
+   */
+   // handle cases if input comes from readDatePinned() function
+   if (ticketDatetimeString == 'Attribute Not Set'){
+      return 'Ticket Timestamp Currently Unavailable'
```

Comment:
in this scenario, i'd prefer to not show anything.  To support that, you probably want to move the "on" part from the Vue template code into this function